### DISCUSSION TITLE
sync issues in crdroid

### DISCUSSION THREAD

**Discussion by git404x** [ROM-builders/temporary#18831] [Q&A]
I was trying to build crdroid ( a12) for my device but it sync keeps on failing :

1.) https://cirrus-ci.com/build/6553145958989824
2.) https://cirrus-ci.com/build/5250049878261760

> Tried 2 times but tsync task fails everytime , Need to fix this

**Comment by git404x** [comment]
@Apon77 Have a Look At the issue plz

**Reply by Apon77** [reply]
> @Apon77 Have a Look At the issue plz

i get notification without tagging

**Reply by git404x** [reply]
Nvm , but what's causing sync to fail? It shows not able to fetch the trees .

![IMG_20230314_075536_963](https://user-images.githubusercontent.com/72090728/224876873-1738108a-eb77-448e-b77f-b53bc4b51cd9.jpg)

**Comment by Apon77** [comment]
issue from your side, fix your local manifest.

**Reply by git404x** [reply]
Ok
